src/panels/config/integrations/integration-panels/zha/zha-config-dashboard.ts

```diff
@@ -32,6 +32,11 @@ export const zhaTabs: PageNavigation[] = [
     path: `/config/zha/groups`,
     iconPath: mdiFolderMultipleOutline,
   },
+  {
+    translationKey: "ui.panel.config.zha.visualization.caption",
+    path: `/config/zha/visualization`,
+    iconPath: mdiFolderMultipleOutline,
```

Comment:
You should not use the same icon.

Reply:
agree... this was left from copying the entry above. Any preference on an icon to use?

Reply:
done